apps/testingaccessibility/src/utils/get-resource-paths.ts

```diff
@@ -0,0 +1,37 @@
+import {SanityDocument} from '@sanity/client'
+import find from 'lodash/find'
+import isEmpty from 'lodash/isEmpty'
+
+export const getPathForLesson = (
```

Comment:
suggestion: these should be simple to tests. rule of thumb is that utils are tested if possible